docs/satellite-update.md

```diff
@@ -1,15 +1,16 @@
 ## Update Satellite to a new minor version
-#### **NOTE: THIS SHOULD ONLY BE USED FOR SATELLITE 6.2.X VERSIONS**
-1. Run the ansible playbook:
+### **Important warnings - read before running this playbook**
+ - This script must be used for Satellite 6.2 only.
```

Comment:
If there is currently only one warning, do we really need a list and separate section?